fairwinds-insights.yaml

```diff
@@ -0,0 +1,11 @@
+options:
```

Comment:
Is there any way we could keep this file somewhere else, like in `.circleci` or as a `.` file?

Reply:
🤔 not currently, but added a `TODO` and created an issue in Insights to make this configurable